lib/dynflow/execution_plan/steps/abstract.rb

```diff
@@ -38,6 +39,8 @@ def initialize(execution_plan_id,
 
         @queue             = Type! queue, Symbol, NilClass
 
+        @wakeup_info       = Type! wakeup_info, Action::Suspended::WakeupInfo if wakeup_info
```

Comment:
looks like `Type! wakeup_info, Action::Suspended::WakeupInfo, NilClass` is the algebrick way